packages/fixie-common/src/agent.ts

```diff
@@ -264,7 +264,7 @@ export class FixieAgentBase {
   }: {
     defaultRuntimeParameters?: Record<string, unknown>;
     externalUrl?: string;
-    runtimeParametersSchema?: string;
+    runtimeParametersSchema?: Record<string, unknown> | null;
```

Comment:
Why can this be `null` ?

Reply:
It was/remains optional; I consolidated with `undefined` instead.